### DISCUSSION TITLE
vllm调用128k的，停不下来呀。后面的都是重复的。

### DISCUSSION THREAD

**Discussion by lvbinandylau** [zai-org/ChatGLM3#986] [General]
vllm调用128k的，停不下来呀。后面的都是重复的。

**Comment by zRzRzRzRzRzRzR** [comment]
special token设置有问题

**Comment by zRzRzRzRzRzRzR** [comment]
把这个配置 https://huggingface.co/THUDM/chatglm3-6b/blob/main/tokenizer_config.json 换到 chatglm3-128k试试

**Comment by majia1984** [comment]
不行，还是一直重复
